packages/charts/src/core/plugins/gauge/linear-gauge-labels-plugin.spec.ts

```diff
@@ -50,7 +50,7 @@ describe("LinearGaugeLabelsPlugin >", () => {
     });
 
     it("should render the same number of threshold labels as there are thresholds", () => {
-        expect(labels.nodes().length).toEqual(Object.keys(gaugeConfig.thresholds as IGaugeThresholdConfigs).length);
+        expect(labels.nodes().length).toEqual(Object.keys(gaugeConfig.thresholds?.definitions as GaugeThresholdDefs).length);
```

Comment:
Is it possible that a reference to an optional `definitions` will get resolved to `undefined`? If yes, In this case `Object.keys` won't be able to process the `undefined` value properly and will throw the TypeError

Reply:
Ah, I see, its in tests. You can ignore the comment, this is trivial in this context